### ISSUE TITLE
Better Tank

### ISSUE THREAD

**Issue by CookerBower** [studio-minus/ppg-bugs-and-requests#1411]
**Expanded Mechanics**
The current tank in ppg is invincible. It needs internal components. cars have internal parts like engines and fuel tanks, and tanks need those too. Other stuff like a functional top hatch and room for humans inside would be cool, but it would be good just with an engine or some weakspot. 

**Roleplaying**
Usable tanks would make roleplaying with the military so much more fun, the RPG already has a feature where it shoots shrapnel on the other side of metal targets, we have the anti-tank but not the tank.

**Comment by Drogo-man** [comment]
This was added in 1.23 tho...

**Comment by CookerBower** [comment]
Yeah i was editing my suggestion rn since I just figured out they do in fact have engines and fuel tanks like cars, but still I feel having the ability to put someone in a tank would be cool
